receiver/README.md

```diff
@@ -151,22 +151,42 @@ receivers:
               - targets: ['localhost:9777']
 ```
 
-### Include Filter
-Include Filter provides ability to filter scraping metrics per target. If a filter is specified for
-a target then only those metrics which exactly matches one of the metrics specified in the `Include Filter` list will be scraped.
-Rest of the metrics from the targets will be dropped.
+### Job Export Prefix
+Job Export Prefix provides the ability to add a metrics prefix per target. If a prefix is specified for a target then
+those metrics are scraped from that target will have the prefix added in the exported name.
 
 #### Syntax
 - Endpoint should be double quoted.
-- Metrics should be specified in form of a list.
+- Prefix should be double quoted with a "/" at the end.
```

Comment:
Looks like none of the examples below has a "/" at the end.

Reply:
Done.